app/decorators/service_provider_session_decorator.rb

```diff
@@ -59,6 +59,10 @@ def sp_name
     sp.friendly_name || sp.agency
   end
 
+  def sp_agency
+    sp.agency || sp.friendly_name
```

Comment:
Can we put something here for SPs that don't have either an agency or friendly name? They should have both, but we don't validate that so it is possible to get into that state.

Reply:
Actually, I'm backtracking. We don't do that for SP name above so maybe it isn't a huge issue.

Reply:
This is the same as the existing `sp_name`, except agency and friendly_name are reversed. Why can't we use the existing `sp_name`, which is the preferred method since we want to use friendly_name first, and then fall back to agency if there isn't one.

Reply:
@monfresh so this is new from @mkhandekar, in the success screen, in the second paragraph we describe the attributes as going back to the agency. https://user-images.githubusercontent.com/1598889/29473964-7a50990c-83f5-11e7-808a-98feb4b842b0.png

Reply:
Ah, I missed that. Thanks for the clarification.